addons/hr_timesheet/views/project_task_views.xml

```diff
@@ -117,21 +117,21 @@
                             <field name="subtask_effective_hours" class="mt-2" widget="timesheet_uom"
                                    attrs="{'invisible' : [('subtask_effective_hours', '=', 0.0)]}" nolabel="1"/>
                             <span attrs="{'invisible': [('subtask_effective_hours', '=', 0.0)]}" class="o_td_label float-start">
```

Comment:
Maybe I'm saying something stupid, but doesn't it work if you put the class on the span?
```suggestion
                            <span attrs="{'invisible': [('subtask_effective_hours', '=', 0.0)]}" class="o_td_label float-start my-1">
```

Reply:
Very true ! Thanks It is better !